src/ign.hh

```diff
@@ -22,7 +22,8 @@
 
 /// \brief External hook to read the library version.
 /// \return C-string representing the version. Ex.: 0.1.2
-extern "C" GZ_FUEL_TOOLS_VISIBLE char *ignitionVersion();
+extern "C" GZ_FUEL_TOOLS_VISIBLE char *gzVersion();
+extern "C" GZ_FUEL_TOOLS_VISIBLE char GZ_DEPRECATED(8) *ignitionVersion();
```

Comment:
No need to tick-tock, this isn't installed

Reply:
https://github.com/gazebosim/gz-fuel-tools/pull/259/commits/59add5051848659b9c291109fafe3b429868b612